src/components/PetsList.jsx

```diff
@@ -5,20 +5,29 @@ import React from "react";
 
 const { getValue } = Config;
 const {
-  pets: { petStatus = console.error("Please define a pet status") || "" } = {},
+  pets: {
+    petStatus = console.error("Please define a pet status") || "",
+    workingCats,
+    informationAbout,
+  } = {},
 } = window;
 
 const PetsList = ({ filters = [] }) => (
-  <FilterList
-    title={`${petStatus} Pets`}
-    filters={filters}
-    apiEndpoint={`${getValue("apiRoot")}`}
-    renderItem={(props) => <PetCard {...props} />}
-    includeInputFilter={true}
-    includeClearButton={true}
-    searchCategory="Pets"
-    inputFilterPlaceholder="Begin typing to filter the results..."
-  />
+  <div>
+    {workingCats === "true" ? (
+      <div dangerouslySetInnerHTML={{ __html: informationAbout }}></div>
```

Comment:
I guess there is no safelySetInnerHTML 😄

Reply:
I've seen this done in configuration and APIs before: the self-documenting "are you sure you know what you're doing!?" method or function.